examples/i2c/host_rpi/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.18)
```

Comment:
I used an older version of CMake because that was the version I had in my Pi. As I followed the instructions in [vocalfusion-rpi-setup](https://github.com/xmos/vocalfusion-rpi-setup/blob/master/README.md), I assumed it is better using a slightly older version of CMake rather than update the setup instructions. We don't need the latest version of CMake for the examples.

Reply:
@lucianomartin I would personally prefer the same version for all the repo, device or host.